.eslintrc.js

```diff
@@ -1,3 +1,8 @@
+const semver = require('semver');
+
+// Converts semver range `~16.6.0` to strict version `16.6.0`
+const reactVersion = semver.coerce(require('./package.json').dependencies.react).raw;
```

Comment:
I wonder what is the difference between `semver.coerce().raw` and `semver.coerce().version`.